internal/lint/pod_test.go

```diff
@@ -133,25 +136,21 @@ func TestPodLint(t *testing.T) {
 	assert.Equal(t, 0, len(ii))
 
 	ii = po.Outcome()["default/p2"]
-	assert.Equal(t, 6, len(ii))
+	assert.Equal(t, 4, len(ii))
 	assert.Equal(t, `[POP-207] Pod is in an unhappy phase ()`, ii[0].Message)
 	assert.Equal(t, `[POP-208] Unmanaged pod detected. Best to use a controller`, ii[1].Message)
-	assert.Equal(t, `[POP-1204] Pod Ingress is not secured by a network policy`, ii[2].Message)
-	assert.Equal(t, `[POP-1204] Pod Egress is not secured by a network policy`, ii[3].Message)
-	assert.Equal(t, `[POP-206] Pod has no associated PodDisruptionBudget`, ii[4].Message)
-	assert.Equal(t, `[POP-301] Connects to API Server? ServiceAccount token is mounted`, ii[5].Message)
+	assert.Equal(t, `[POP-206] Pod has no associated PodDisruptionBudget`, ii[2].Message)
+	assert.Equal(t, `[POP-301] Connects to API Server? ServiceAccount token is mounted`, ii[3].Message)
 
 	ii = po.Outcome()["default/p3"]
```

Comment:
`np2` has no `podSelector` (this is being reported as an invalid schema).. assuming that's the same as an empty `podSelector`, which should apply to all pods in the namespace then. 
`p3` is in the `default` namespace, and `np2` has both ingress and egress policies, so therefore `p3` is secured by ingress & egress,